External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -4005,6 +4005,24 @@ void OpDispatchBuilder::MOVMSKOp(OpcodeArgs) {
   StoreResult(GPRClass, Op, CurrentVal, -1);
 }
 
+void OpDispatchBuilder::MOVMSKOpOne(OpcodeArgs) {
+  OrderedNode *CurrentVal = _Constant(0);
+  OrderedNode *Src = LoadSource(FPRClass, Op, Op->Src[0], Op->Flags, -1);
+
+  auto M = _Constant(0x80'40'20'10'08'04'02'01ULL);
+  OrderedNode *VMask = _VCastFromGPR(16, 8, M);
```

Comment:
Might be nice to implement a `VDUPFromGPR` IR op to remove this VCastFromGPR+VInsGPR pair.
This would map directly to AArch64's dup instruction and remove another instruction from this emulation.

Either add a comment about this minor opt and open and issue to track it, or do it it in this PR, making sure it is implemented in all three backends.
I'm fine with either route, otherwise looks good.

Reply:
Will add a comment, I think adding an additional opcode is most likely out-of-scope here.